ssl/t1_lib.c

```diff
@@ -3155,7 +3155,7 @@ static const SIGALG_LOOKUP *find_sig_alg(SSL *s, X509 *x, EVP_PKEY *pkey)
         if (lu->sig == EVP_PKEY_EC) {
 #ifndef OPENSSL_NO_EC
```

Comment:
Should not it be possible to remove this ifndef now?

Reply:
Done